commands/core/init.drush.inc

```diff
@@ -0,0 +1,115 @@
+<?php
+
+/**
+ * @file
+ *   Set up local Drush configuration.
+ */
+
+/**
+ * Implementation of hook_drush_command().
+ *
+ * @return
+ *   An associative array describing your command(s).
+ */
+function init_drush_command() {
+  $items['core-init'] = array(
+    'description' => 'Initialize local Drush configuration.',
+    'aliases' => array('init'),
+    'bootstrap' => DRUSH_BOOTSTRAP_NONE,
```

Comment:
Yes, it would work okay, but only if Drush was already installed remotely. It seems that a local call to `drush init` after installation is going to be the typical use-case. Even if the user is trying to "catch up" and init a bunch of remote Drush installations, `drush @remote ssh` followed by `drush init` is a more reasonable approach.

Another issue about `drush @remote init` that might be strange is that the remote version of Drush might be different than the local version of Drush.  Right now, the operations performed by `drush init` should work all the way back to Drush 5; I don't know how likely it is that we would make `drush init` set something up that only worked on the most recent version of Drush, but I am all the same reluctant to promote remote execution of init.